### DISCUSSION TITLE
Add DDNS support to Dashboard

### DISCUSSION THREAD

**Discussion by Swayz3** [gameserverapp/Platform#568] [Suggestions]
Dynamic Domain Name System (DDNS) is a game-changer in the world of networking, and here's why you should consider embracing it. DDNS offers you the flexibility and convenience of accessing your devices or services remotely with ease. No more struggling to remember complex IP addresses or dealing with the hassle of manual updates.

With DDNS, your devices are assigned a domain name that remains constant, even if your IP address changes. This means you can effortlessly connect to your home security cameras, servers, or other devices from anywhere in the world, without the need for a static IP address. It's a seamless solution that brings convenience to the forefront, ensuring you stay connected without the headaches.

It would be very appreciated to have DDNS support on the dashboard for Dediconnect machines.

**Comment by gnarlyvr1** [comment]
DDNS would make self-hosting accessible for users who are unable to get a static IP from their ISP.

**Comment by mvaessen** [comment]
Going to reject this. It's not desirable because it can be used for attacking GSA. Entering an IP will be required.
I understand that this make it less interesting for folks hosting from home.
